ubyssey/templates/widgets/frontpage/facebook-video-big.html

```diff
@@ -0,0 +1,48 @@
+{% if do_show %}
+<div id="fb-root"></div>
+<div class="c-facebook-video">
+{% if title %}
+	<h1>{{ title }}</h1>
+{% endif %}
+
+	<div class="c-facebook-video__info">
+	{% if host %}
+		<span class="c-facebook-video__host">By {{ host }}</span>
+	{% endif %}
+	{% if start_time %}
+		<span>&nbsp;·&nbsp;</span>
+		<span class="c-facebook-video__start-time">{{ start_time }}</span>
+		{% endif %}
+	{% if end_time %}
+		<span class="c-facebook-video__end-time">until
+		{% if start_time|date:"Ymd" == end_time|date:"Ymd" %}
+			{{ end_time|date:"g:i a" }}
+		{% else %}
+			{{ end_time }}
+		{% endif %}
+		</span>
+	{% endif %}
+	{% if description %}
+		<div class="c-facebook-video__description">{{ description }}</div>
+	{% endif %}
+	</div>
+
+	<div class="c-facebook-video__container">
+		<div class="fb-video" data-href="{{ video_url }}" data-show-text="false"></div>
+
+		{% if show_comments %}
+			<div id="comments-mount">
+				<div class="fb-comments" data-href="{{ video_url }}" data-width="350" data-numposts="5"></div>
```

Comment:
Maybe reduce `numposts` to 3 and set overflow to `scroll` to prevent this element from growing too tall